internal/ci/ethereum_test

```diff
@@ -69,10 +69,24 @@ trap "exit_handler" EXIT SIGTERM SIGINT
 
 pushd $SRCROOT >/dev/null
 
-# Install gethnet 1.8.8
+########################
+## Retrieve gethnet version from Gopkg.lock
+########################
+
+printf -- "\033[34mRetrieving geth version...\033[0m\n"
+go install github.com/pelletier/go-toml/cmd/tomljson
```

Comment:
Could we put this in the builder image?

Reply:
We could, but then people would run into issues if they ran it locally. The speed cost is a few seconds.

I also wanted to ask if we should fork this into a `smartcontractkit` repo.